package.json

```diff
@@ -50,6 +50,8 @@
     "prop-types": "^15.6.1",
     "react-icon-base": "^2.1.2",
     "react-icons": "^2.2.7",
+    "styled-components": "^3.2.6",
+    "styled-jsx": "^2.2.6",
```

Comment:
`styled-jsx` is already in peer & dev dependencies, pls don't add it here

Reply:
Fixed.